src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js

```diff
@@ -230,7 +230,7 @@ export const GenericTable = ({
 
     const agGridData = _generateGridDataFromCSV(fileContent, parameterData, options);
     if (agGridData.error) {
-      if (clientFileDescriptorRestoreValue) {
+      if (clientFileDescriptorRestoreValue?.id) {
```

Comment:
As discussed, we should probably check the `status` instead of the `id` (with an optional chaining operator in case the object is undefined: `?.status`)

Reply:
DONE